dataprofiler/profilers/profile_builder.py

```diff
@@ -1519,7 +1520,10 @@ def __init__(
         # Structured specific properties
         self.row_has_null_count = 0
         self.row_is_null_count = 0
-        self.hashed_row_dict: dict = dict()
+        if options.hll_row_hashing:
```

Comment:
to avoid the if else, set it to a dict and when the if condition hits, it will overwrite it.

Reply:
I wonder if `hll_row_hashing` is a good name. ppl might not know what it is. Anyway we could refer to unique row count and this is probabilistic. If no good options, keep.